cs3api4lab/handlers.py

```diff
@@ -1,14 +1,26 @@
+import os
+import json
+
+from notebook.base.handlers import APIHandler
```

Comment:
There's another thing that I forgot to mention.. You made this extension compatible with jupyter-server. But then you are using `notebook` and retrieving its methods+classes. While this should be ok for now, it might not be in the future so we could take the opportunity to improve the dependencies. I would remove notebook from the dependencies completely.

Reply:
Sure we can do that, but I am a bit confused because the examples https://github.com/jupyterlab/extension-examples are Jupyter 3 compatible and still use notebook imports

Reply:
Let's create a ticket and do that after, then, if ok with you.

Reply:
https://github.com/sciencemesh/cs3api4lab/issues/63